routes/milestones.js

```diff
@@ -1,9 +1,11 @@
 var express = require("express");
 var router = express.Router();
+const db = require("../model/helper");
 
 /* GET home page. */
 router.get("/", function (req, res, next) {
```

Comment:
I'd encourage you to use modern JavaScript syntax here to keep things concise and readable. Are you familiar with arrow functions?

Rewritten this would look like this: 

```
router.get("/", (req, res, next) => {
  res.send({ title: "Express" });
});
```

Reply:
Updated.